ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -43,9 +44,13 @@ const TableCell = styled(Table.Cell)`
   vertical-align: top;
 `
 
+const TitleTableCell = styled(TableCell)`
```

Comment:
I am concerned that you are unconditonally changing the formatting for this everywhere this component is used. Why are these changes neccesssary?

Reply:
Basically, it adds styling to the table cell width. Without the styling, the table looks weird as below.
The bulk-edit-individual modal:
![image](https://user-images.githubusercontent.com/33550059/105763818-30f8b880-5f24-11eb-9d1f-5ccab029bcc1.png)
The new create-project-from-anvil form:
![image](https://user-images.githubusercontent.com/33550059/105763988-61d8ed80-5f24-11eb-8965-f4c087a79c87.png)

Reply:
Oh I see, and yeah that is something we would want to fix everywhere. I think a better way of fixing it, though, is to instead of having 3 different Table rendered here, we could use one big table, and then the columns would all be aligned properly on every row. You can use the `colSpan` property on a cell to make it use multiple columns, so you can do that for the description text rows that can use the full width

Reply:
Gotcha, but I maybe do it a little later.

Reply:
I actually ended up needing to do this for an unrelated PR that updates the bulk upload for IGV, so if you want an example you can see that here

https://github.com/broadinstitute/seqr/commit/0baa4a95a3fa4e6c9f80f2cb322b901b6aeaa91a#diff-b7544cdb2ed88c5c4972f5e29449806f151c89bafbfe26c3f3b0852dea4f9ad8R93